opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -61,18 +61,31 @@ message MetricDescriptor {
   string unit = 3;
 
   // The kind of metric. It describes how the data is reported.
+  //
+  // A gauge is an instantaneous measurement of a value.
+  //
+  // A cumulative measurement is a value accumulated over a time interval. In
+  // a time series, cumulative measurements should have the same start time and
+  // increasing end times, until an event resets the cumulative value to zero
```

Comment:
nit: there are int64, double and distribution cumulative values, and "resets to zero" probably doesn't apply to distribution values. maybe remove "to zero"?

Reply:
Distributions do get reset to zero, e.g. on program start.